CommentAgent.py

```diff
@@ -170,7 +170,9 @@ def parse_python_code(self, llm_answer):
             print("Found Python Block: ", python_code_block)
             return python_code_block
         else:
-            raise ParsingError("No python snippet provided")
+            raise ParsingError(
+                "No python snippet provided, remember to use ```python\n<valid_python_code>\n```"
+            )
```

Comment:
The exception message can be made more descriptive to guide the user effectively.
        ```suggestion
        raise ParsingError(
            "Python code snippet not detected. Ensure your response includes a valid Python "
            "code block enclosed within triple backticks and starting with ```python."
        )
        ```